kitchen-sink/react/pages/ContactsList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  Page,
+  Navbar,
+  NavbarBackLink,
+  List,
+  ListGroup,
+  ListItem,
+} from 'konsta/react';
+
+export default function ContactsListPage() {
+  const isPreview = document.location.href.includes('examplePreview');
+  return (
+    <Page>
+      <Navbar
+        title="Contacts List"
+        left={!isPreview && <NavbarBackLink onClick={() => history.back()} />}
+      />
+      <List strongIos>
+        <ListGroup>
+          <ListItem title="A" groupTitle contacts="true" />
+          <ListItem title="Aaron" dividers={false} contacts="true" />
+          <ListItem title="Abbie" dividers={false} contacts="true" />
```

Comment:
Remove dividers={false} everywhere, ListItem can inherit this property from ListGroup where you should set it to false